### ISSUE TITLE
[bug] compatibility for windows 10

### ISSUE THREAD

**Issue by ain-soph** [hagenw/sphinxcontrib-katex#29]
I suffer from this bug when compiling Pytorch Documents on Windows System.

https://github.com/hagenw/sphinxcontrib-katex/blob/88da3b08eb85cfc8eb126da62f4b1e7db39eed8c/sphinxcontrib/katex.py#L79-L83

leads to the Exception:
`FileNotFoundError: [WinError 2] The system cannot find the file specified`

This code doesn't work for Windows System because `'katex'` can't be found even though it is in $PATH$ and can be executed in command line.

You can view the solution here: https://stackoverflow.com/questions/5658622/python-subprocess-popen-environment-path

> A simple way that should work on Windows too is to explicitly give `os.environ['PATH']` as the argument `env` to `subprocess.Popen`

Example Solution (The program is `katex.cmd` on Windows rather than `katex` on Linux. Might need to deal with that for compatibility.)
```python
    p = subprocess.Popen(
        ('katex.cmd', ) + options,
        stdout=subprocess.PIPE,
        stdin=subprocess.PIPE,
        env=os.environ.copy()
    )
```

**Comment by hagenw** [comment]
Thanks for reporting, I will fix this on Monday.
  

  

  

  
  
  
  
  
>

**Comment by hagenw** [comment]
I tried to fix it with #30 and released version 0.6.1. Could you please try if it now works under Windows. During the next days I will also try to add a CI job under Windows to test it directly.

**Comment by ain-soph** [comment]
> I tried to fix it with #30 and released version 0.6.1. Could you please try if it now works under Windows. During the next days I will also try to add a CI job under Windows to test it directly.

I've checked and it's good on my side.
